pkg/workloads/console/controller.go

```diff
@@ -373,7 +409,24 @@ func (r *reconciler) clampTimeout(template *workloadsv1alpha1.ConsoleTemplate) e
 	return nil
 }
 
-func (r *reconciler) updateStatus(job *batchv1.Job) error {
+func (r *reconciler) authorised(rule *workloadsv1alpha1.ConsoleAuthorisationRule) (bool, error) {
+	if rule == nil {
+		return true, nil
+	}
+
+	auth, err := r.getConsoleAuthorisation()
+	if err != nil {
+		return false, err
+	}
+
+	if len(auth.Spec.Authorisations) == rule.ConsoleAuthorisers.AuthorisationsRequired {
```

Comment:
To protect against two people simultaneously authorising; or someone adding an additional authorisation after the console has been created?

```suggestion
	if len(auth.Spec.Authorisations) >= rule.ConsoleAuthorisers.AuthorisationsRequired {
```